### PR TITLE
Ff buttons fix

### PR DESCRIPTION
**Pull request by cyraxjoe**
Rather _dirty_ fix for the problem with unresponsive buttons for firefox in the dashboard. 

Also... download the background image used in the dashboard. I see no reason to download it from the other server. But correct me if I'm wrong.

### CODE REVIEW

File: static/css/dashboard.css [hunk 1]
```diff
@@ -207,7 +207,7 @@ body{
 
 .ticket-details{
   background:white;
-  background-image:url('http://trevorjwelch.com/imgs/Border001.png');
+  background-image:url('/static/img/Border001.png');
```
Comment (czpython) [inline static/css/dashboard.css]: This image doesn't exist on server.
All media files (css, js, img) get a hash prepended to their filename.
Could you add this background-image to the html directly?
Then we can do `{% static 'img/Border001.png' %}`

**Review by czpython — CHANGES_REQUESTED** [review]

File: static/css/dashboard.css (hunk 1 above)
Comment (cyraxjoe) [reply static/css/dashboard.css]: Sure.

Comment (cyraxjoe) [reply static/css/dashboard.css]: Move it as an in-line style, right after the css import in the dashboard.

**Review by czpython — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**